src/MissionManager/AviantMissionTools.cc

```diff
@@ -55,12 +55,22 @@ QUrl AviantMissionTools::_getUrl(Operation operation, QString base)
         case RallyPointHeight:
             return QUrl(base + "/set_rally_points_height");
             break;
+        case FetchKyteOrders:
+            return QUrl(base + "/orders/api/v2/orders/active/"); 
         case NoOperation:
         default:
             return QUrl();
     }
 }
 
+QUrl AviantMissionTools::_getUrl(Operation operation, QString base, int orderId) {
+    if (operation == FetchKyteOrderMissionFile) {
+        return QUrl(base + "/orders/api/v2/orders/" + QString::number(orderId) + "/mission-file/");
+    } else {
+        return _getUrl(operation, base);
+    }
+}
+
```

Comment:
mixing kyte_backend and mms calls in this way is confusing

Reply:
I created separate _getMmsUrl and _getKyteBackendUrl methods